util/config_file.c

```diff
@@ -543,6 +544,7 @@ int config_set_option(struct config_file* cfg, const char* opt,
 	else S_NUMBER_NONZERO("tcp-reuse-timeout:", tcp_reuse_timeout)
 	else S_YNO("edns-tcp-keepalive:", do_tcp_keepalive)
 	else S_NUMBER_NONZERO("edns-tcp-keepalive-timeout:", tcp_keepalive_timeout)
+	else S_NUMBER_NONZERO("sock-queue-timeout:", sock_queue_timeout)
```

Comment:
With this change it matches the conf parser, where 0 is accepted.
```suggestion
	else S_NUMBER_OR_ZERO("sock-queue-timeout:", sock_queue_timeout)
```

Reply:
Ah, yeah, you are right, missed this part, thanks!